core/domain/opportunity_services.py

```diff
@@ -147,9 +147,12 @@ def _create_exploration_opportunity_summary(topic, story, exploration):
     """
     language_codes_needing_voice_artists = set(
         exploration.get_languages_with_complete_translation())
-    # Add exploration language to voiceover required languages list as an
-    # exploration can be voiceovered in its own language.
-    language_codes_needing_voice_artists.add(exploration.language_code)
+    incomplete_translation_language_codes = (
```

Comment:
Could you please add a test case that would have caught the original error?

Reply:
I think this logic needs to be revisited anyhow if/when voiceover opportunity support is added, as it doesn't seem correct that the translation status of voiceovers should depend on the written translation status?

Reply:
Yeah, I agree, this seems a bit weird to me and I'm confused by the logic. The only dependency I'm aware of is that voiceovers can only be done when the translations are done (typically) and I think that was what the previous code was (not very well) trying to model.

Maybe keep it as it used to be, for now?